theme/management/commands/add_quota_avu.py

```diff
@@ -30,6 +34,16 @@ def handle(self, *args, **options):
                             print res.short_id + ' does not have an owner'
                             continue
                     res.set_quota_holder(res.creator, res.creator)
+
+                # Now reset the quota system by setting the AVU 'resetQuotaDir' to anything
+                istorage = res.get_irods_storage()
+                # Add a specific AVU to invoke quota recalculation
+                istorage.session.run("imeta", None, 'set', '-C', res.root_path,
+                                     'resetQuotaDir', '0')
+                # Now remove that AVU to clean up; this is not part of the resource API
+                istorage.session.run("imeta", None, 'rm', '-C', res.root_path,
+                                     'resetQuotaDir', '0')

```

Comment:
@hyi : 

+ Is this new avu 'resetQuotaDir' used to invoke the PEP micro-service to calculate quota usage for the quota holder? 
Yes, but not only. @mjstealey could update his nightly script a litte bit to do that.

+ Where is the quota usage for each quota holder stored? 
Quota usage of all quota holders are stored as AVUs of HydroShare Proxy User. Each HydroShare user such as pdoan would have two AVUs: pdoan-quota & pdoan-usage (the unit of these values is Bytes). If pdoan-quota is not set or have 0 value => this user is unlimited.